asset/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import FileBatcher from '../src/file_exporter/processor';
+import Schema from '../src/file_exporter/schema';
+
+import FileFetcher from '../src/file_reader/fetcher';
+import FileSlicerOperation from '../src/file_reader/slicer';
+import Schema from '../src/file_reader/schema';
```

Comment:
The `Schema` objects in both the `file_exporter` and `file_reader` are named `Schema` and don't currently have unique names.  Can I rename them using `as` or something?  Or will this somehow result in trouble?

Reply:
I talked with Peter about this, I've finished this with his recommendations.